tests/behat/behat_repository_ocis.php

```diff
@@ -152,6 +184,12 @@ private function delete_file_in_personal_space($user, $file) {
         }
     }
 
+    /**
+     * Delete created users
```

Comment:
```suggestion
     * Delete a user
```

this function will only delete one user, and even if the user is not created by the tests.
So maybe also rename the function to `delete_user`